manifests/init.pp

```diff
@@ -13,6 +13,8 @@
   String $conf_d,
   String $confdir,
   String $config_file,
+  String $domain_insecure,
```

Comment:
In order to allow the `~` option above, this will need to change to `Optional[String] $domain_insecure`.